website/docs/how-to/using-a-capacitor-plugin.md

```diff
@@ -2,3 +2,276 @@
 title: How To Use a Capacitor Plugin
 sidebar_label: Use a Capacitor Plugin
 ---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Ionic Portals uses Capacitor under the hood, meaning that you can use existing Capacitor Plugins in your Portals. These plugins allow Portals to use native functionality without much setup on either the native or web developers part.
```

Comment:
might want to say "existing Capacitor Core plugins" with a link to either a doc here or the core plugins page in the cap docs